packages/ckeditor5-font/src/ui/colortableview.ts

```diff
@@ -391,6 +433,81 @@ export default class ColorTableView extends View {
 		return buttonView;
 	}
 
+	/**
+	 * Creates bar with "save" and "cancel" buttons in it.
+	 */
+	private _createAtionBarView( { saveButtonView, cancelButtonView }: {
+		saveButtonView: ButtonView;
+		cancelButtonView: ButtonView;
+	} ): View {
+		const actionBarRow = new View();
+		const children = this.createCollection();
+
+		children.add( saveButtonView );
+		children.add( cancelButtonView );
+
+		actionBarRow.setTemplate( {
+			tag: 'div',
+			attributes: {
+				class: [
+					'ck',
+					'ck-color-table_action-bar'
+				]
+			},
+			children
+		} );
+
+		return actionBarRow;
+	}
+
+	/**
+	 * Creates "save" and "cancel" buttons.
+	 */
+	private _createActionButtons() {
+		const locale = this.locale;
+		const saveButtonView = new ButtonView( locale );
+		const cancelButtonView = new ButtonView( locale );
+
+		saveButtonView.set( {
+			icon: icons.check,
+			class: 'ck-button-save',
+			type: 'submit'
+		} );
+
+		cancelButtonView.set( {
+			icon: icons.cancel,
+			class: 'ck-button-cancel'
```

Comment:
We need a `label` for accessibility. It can be "hidden" to show only the icon - use `withText: false`